### ISSUE TITLE
Invalid repo token on calling `done` api

### ISSUE THREAD

**Issue by sashaaKr** [lemurheavy/coveralls-public#1750]
TL;DR

We are having a problem calling `coveralls done` that results with `{"error":"Invalid repo token"}`

More information:

We have an external from github actions CI/CD system that runs the tests and report coverage to coveralls using coveralls cli, version `0.6.6`. 

CI/CD uses a github application that is installed on github at the organization level and has access to all repos within the organization. This allows it to perform all necessary github operations, such as reporting PR status checks.

We are using [Create an installation access token for an app](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app) api in order to create access token, similarly to how it works with coveralls github action that [using](https://github.com/coverallsapp/github-action/blob/main/action.yml#L6) `GITHUB_TOKEN`.

We are using the generated token as `COVERALLS_REPO_TOKEN`, and by using this token we can perform the `coveralls report` command, however we are failing to run `coveralls report` which results in api response `{"error":"Invalid repo token"}`.

We tried to extend application permissions, and granted write permission to different scopes (with much broader scopes than [GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) has), however with no success.

Will appreciate your help to understand if there is any specific scope that needs to be granted to GITHUB_TOKEN to run the 'done' command?

Or is it possible we discovered the bug on the coveralls side?

**Comment by levan-giguashvili-lmnd** [comment]
Folks, can we please get some response here? or a workaround so that we can simplify our solution

**Comment by afinetooth** [comment]
@levan-giguashvili-lmnd Can you please share:

1. The Coveralls URL to one of your unfinished builds?
2. Your GitHub Actions config pertaining to Coveralls steps?
